src/features/user/TreeMapper/Analytics/components/Export/index.tsx

```diff
@@ -0,0 +1,264 @@
+    const worksheet = utils.json_to_sheet(data);
+    const workbook = utils.book_new();
+
+    const descWorksheet = utils.json_to_sheet(
+      description.map((d) => ({
+        column_title: d.title,
+        description: d.description,
+      }))
+    );
+    utils.book_append_sheet(workbook, worksheet, getSheetTitle());
+    utils.book_append_sheet(workbook, descWorksheet, t('readme'));
+    const xlsxBuffer = write(workbook, {
+      bookType: 'xlsx',
+      type: 'array',
+    });
+
+    const blob = new Blob([xlsxBuffer], {
+      type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+    });
+    saveAs(
+      blob,
+      `${localProject?.name}__${format(localFromDate, 'dd-MMM-yy')}__${format(
+        localToDate,
+        'dd-MMM-yy'
+      )}`
+    );
+  };
+
+  const handleExport = async () => {
+    if (localProject) {
+      // TODO - Once error handling PR is merged refactor this fetch call with a makeNextRequest function
+
+      const res = await fetch('/api/data-explorer/export', {
+        method: 'POST',
+        body: JSON.stringify({
+          projectId: localProject.id,
+          startDate: localFromDate,
+          endDate: localToDate,
+        }),
+      });
+
+      if (res.status === 429) {
+        handleError({ message: t('errors.tooManyRequest'), type: 'error' });
+        return;
+      }
+
+      const { data } = await res.json();
+
+      if (data.length === 0) {
+        handleError({ message: t('errors.emptyExportData'), type: 'error' });
+        return;
+      }
+
+      extractDataToXlsx(data);
+    }
+  };
+
+  const handleProjectChange = (proj: Project | null) => {
+    setLocalProject(proj);
+  };
+
+  const handleProjectTypeChange = (projType: ProjectType | null) => {
+    setProjectType(projType);
+  };
+
+  return ready ? (
+    <Container title={t('exportData')}>
+      <div className={styles.container}>
+        <ProjectTypeSelector
+          handleProjectTypeChange={handleProjectTypeChange}
+        />
+
+        <ProjectSelectAutocomplete
+          projectList={projectList || []}
+          project={project}
+          handleProjectChange={handleProjectChange}
+        />
+
+        <div className={styles.datePickerContainer}>
+          <LocalizationProvider
+            dateAdapter={AdapterDateFns}
+            locale={
+              localeMapForDate[userLang]
+                ? localeMapForDate[userLang]
+                : localeMapForDate['en']
+            }
+          >
+            <MuiDatePicker
+              label={t('treemapperAnalytics:from')}
+              value={localFromDate}
+              onChange={setLocalFromDate}
+              renderInput={(props) => (
+                <MaterialTextField variant="outlined" {...props} />
+              )}
+              inputFormat="MMMM d, yyyy"
+              maxDate={new Date()}
+              DialogProps={{
+                sx: dialogSx,
+              }}
+            />
```

Comment:
@mohitb35  This is something that needs to be done in other PR, currently we are using the current styles for Date Field everywhere in the project so having newer UX only for analytics will put app out of sync also we need to finalise newer UX so it will be in sync with other Date Field.